### PR TITLE
Docs/adding data

### LINKED ISSUE #166: [DOC]: How to add custom parcellations and friends?
### Which element of the documentation do you want to modify and why?

I would like to add my custom parcellation but the docs don't tell you exactly how. So, would be great to have an example.

### Anything else to say?

_No response_

### PR DESCRIPTION
**Pull request by LeSasse**
* [x] fix #166 
* [x] description of feature/fix
* [x] tests added/passed
* [x] add an entry to the [latest changes](../docs/changes/latest.inc)

Add documentation on how to register custom parcellations, coordinates, and masks.

### CODE REVIEW

File: docs/extending/coordinates.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+this can be done:
+
+.. code-block:: yaml
+
+
+	markers:
+	  - name: DMNCustom_mean
+	    kind: SphereAggregation
+	    coords: DMNCustom
+		radius: 5
```
[the remark this answers is not in the document]
Comment (LeSasse) [reply docs/extending/coordinates.rst:124]: ah damn, is there something like an .rst linter to deal with this type of stuff, @synchon ?

File: docs/extending/coordinates.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+spheres to aggregate (for example using the 
+:class:`junifer.markers.sphere_aggregation.SphereAggregation` marker) the MR
+signals from individual voxels. Now, before you start specifying your own sets
+of coordinates, check the coordinates that Junifer already has
+:ref:`built in already<builtin>`. If you simply want to use a well known set of
+coordinates from the literature, there is a reasonable chance, that Junifer provides
+them.
+
+If you checked the in-built coordinates, and they are not there already (
+for example if you came up with your own set of coordinates), then Junifer provides
```
Comment (synchon) [inline docs/extending/coordinates.rst:20]: Has an extra whitespace between `(` and `for`.

File: docs/extending/coordinates.rst [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+Step 2: Add coordinate registration to the YAML file
+----------------------------------------------------
+
+In order to register your coordinates for a pipeline configured by a YAML file,
+you can use the ``with`` keyword provided by Junifer:
+
+.. code-block:: yaml
+
+	with:
+		- registering_my_parcellation.py
```
Comment (synchon) [inline docs/extending/coordinates.rst:108]: This line again has the extra tab.

File: docs/extending/masks.rst [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+of masks :ref:`in-built in Junifer already<builtin>`, so check if any of them
+suit your needs. Check how to use these masks :ref:`here<using_masks>`. Once you
+know how to use these masks, and you checked whether the in-built masks suit your
+needs, and you have found that they don't, you can come back here to learn how
+to use your own masks.
+
+The principle is fairly simple and quite similar to :ref:`adding_parcellations`
+and :ref:`adding_coordinates`. Junifer provides a :func:`junifer.data.masks.register_mask`
+function that lets you register your own custom masks. It consists of two positional
+arguments (``name`` and ``mask_path``) and one optional key word argument
```
Comment (synchon) [inline docs/extending/masks.rst:19]: `key word` => `keyword`

File: docs/extending/masks.rst [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+Step 2: Configure a YAML file for registration of a mask
+--------------------------------------------------------
+
+In order to do this, we can use the ``with`` keyword provided by Junifer:
+
+.. code-block:: yaml
+
+	with:
+		- register_custom_mask.py
```
Comment (synchon) [inline docs/extending/masks.rst:58]: Extra tab here.

**Review by synchon — CHANGES_REQUESTED** [review]

**Pull request merged**